grove/grovetccfg/grovetccfg.go

```diff
@@ -43,9 +43,6 @@ import (
 	"github.com/apache/trafficcontrol/grove/web"
 )
 
-// Duplicating Hdr and ModHdrs here for now...
-// Seems cleaner than dragging it up from some arbitrary place in plugins
-
 const Version = "0.1"
```

Comment:
@rob05c could you change the version to "0.2".  I changed the rpm build scripts and grove to "0.2" last November and missed changing it in grovetccfg.  the rpm build script for grovetccfg is using version "0.2".

Reply:
Done